_includes/footer.html

```diff
@@ -22,6 +22,28 @@
         {%- include author-links.html author=_site_author -%}
       </div>
     </div>
+    
+    <style type="text/css">
+      td
```

Comment:
If this is only for the footer I think it's better to add a class as well (and then you can probably move the `.custom_font` rules here as well).